app/pages/resource/reservation-info/ReservationInfo.js

```diff
@@ -117,10 +117,20 @@ function renderReservationPeriodInfo(resource, t) {
   );
 }
 
+function renderRequiresManualConfirmationText(t) {
+  return (
+    <p className="needs-manual-confirmation-text">
+      <img alt="" className="app-ResourceHeader__info-icon" src={iconClock} />
```

Comment:
Probably should be an alt text here for accessibility.

Reply:
Good point